examples/kong-ingress.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: configuration.konghq.com/v1
+kind: KongIngress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: kongtests
+  name: kic-event-hooks
+  namespace: kong-system
+
+eventhooks:
+  event: "workspaces"
+  config:
+    url: "https://webhook.site/ec707ef0-ab91-4693-8dd2-114471ff6f90"
```

Comment:
Do not link to real external sites in examples. Use something in the `.example` domain; see [RFC 2606](https://datatracker.ietf.org/doc/html/rfc2606#page-2).